cypress/integration/logistics/materialentnahme_partial_HU.js

```diff
@@ -0,0 +1,244 @@
+import { PackingInstructionsVersion } from '../../support/utils/packing_instructions_version';
+import { Builder } from '../../support/utils/builder';
+import { appendHumanReadableNow } from '../../support/utils/utils';
+import { PurchaseOrder, PurchaseOrderLine } from '../../support/utils/purchase_order';
+import {
+  applyFilters,
+  selectNotFrequentFilterWidget,
+  toggleNotFrequentFilters,
+  clearNotFrequentFilters,
+} from '../../support/functions';
+
+let productForPackingMaterialTU;
+let productForPackingMaterialLU;
+let packingMaterialForTU;
+let packingMaterialForLU;
+let packingInstructionsNameForTU;
+let packingInstructionsNameForLU;
+let productName1;
+let productCategoryName;
+let discountSchemaName;
+let priceSystemName;
+let priceListName;
+let priceListVersionName;
+let productType;
+let vendorName;
+let materialentnahme;
+let warehouseName;
+let packingInstructionsVersionForTU;
+let packingInstructionsVersionForLU;
+
+it('Read the fixture', function() {
+  cy.fixture('logistics/materialentnahme_partial_HU.json').then(f => {
+    productForPackingMaterialTU = appendHumanReadableNow(f['productForPackingMaterialTU']);
+    productForPackingMaterialLU = appendHumanReadableNow(f['productForPackingMaterialLU']);
+    packingMaterialForTU = appendHumanReadableNow(f['packingMaterialForTU']);
+    packingMaterialForLU = appendHumanReadableNow(f['packingMaterialForLU']);
+    packingInstructionsNameForTU = appendHumanReadableNow(f['packingInstructionsNameForTU']);
+    packingInstructionsNameForLU = appendHumanReadableNow(f['packingInstructionsNameForLU']);
+    productName1 = appendHumanReadableNow(f['productName1']);
+    productCategoryName = appendHumanReadableNow(f['productCategoryName']);
+    discountSchemaName = appendHumanReadableNow(f['discountSchemaName']);
+    priceSystemName = appendHumanReadableNow(f['priceSystemName']);
+    priceListName = appendHumanReadableNow(f['']);
+    priceListVersionName = appendHumanReadableNow(f['priceListVersionName']);
+    productType = f['productType'];
+    vendorName = appendHumanReadableNow(f['vendorName']);
+    materialentnahme = f['materialentnahme'];
+    warehouseName = f['warehouseName'];
+    packingInstructionsVersionForTU = appendHumanReadableNow(f['packingInstructionsVersionForTU']);
+    packingInstructionsVersionForLU = appendHumanReadableNow(f['packingInstructionsVersionForLU']);
+  });
+});
+
+describe('Partial material withdrawal in handling unit editor with Materialentnahmelager', function() {
+  it('Create price entities', function() {
+    Builder.createBasicPriceEntities(priceSystemName, priceListVersionName, priceListName, false);
+    cy.fixture('discount/discountschema.json').then(discountSchemaJson => {
+      Object.assign(new DiscountSchema(), discountSchemaJson)
+        .setName(discountSchemaName)
+        .apply();
+    });
+  });
+  it('create product from 24_Gebinde category in order to use it for packing material for transport unit', function() {
```

Comment:
In my tests i never used `24_Gebinde` for packaging materials. Should i also do that? Is there "special usage" for this category. other than the naming? (i just used the product category used for all the other products in my test )

Reply:
The product you want to use as a packing item needs to have the '24_Gebinde' category. Setting the right category in that case is important for accounting, because the product category relates to the cost center. I guess it will work with another category (e.g. 02_Gemuse) and will not pop up as an error, but the customer will always use 24_Gebinde category, that's why it's important to also have it like this in the test. I've checked the information with @metas-dh

Reply:
Thanks for the heads-up. Will use it from now on!